src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -135,14 +136,19 @@ public void AllowReadStreamBuffering_SetTrueThenGet_ExpectTrue(Uri remoteServer)
             }               
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public void ContentLength_SetAfterRequestSubmitted_ThrowsInvalidOperationException(Uri remoteServer)
+        [Fact]
+        public async Task ContentLength_SetAfterRequestSubmitted_ThrowsInvalidOperationException()
```

Comment:
It's good to switch to a loopback, but why did this "need" to be done?  How does it relate to the deadlocks you mentioned?  If the original code caused deadlocks, that's a real problem.